### PR TITLE
Fix navigation + add markdown section permalink things

### PR DESCRIPTION
**Pull request by alazymeme**

### CODE REVIEW

**Comment by fourtf** [comment]
Screenshot at least?

File: mkdocs.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ theme:
   palette:
     scheme: preference
   features:
-    - instant
+    - navigation.instant
```
Comment (alazymeme) [inline mkdocs.yml:9]: This fixes the navigation not being instant - fairly sure the docs might've changed or something.

File: mkdocs.yml [hunk 2]
```diff
@@ -16,6 +16,9 @@ plugins:
       fallback_to_build_date: true
 repo_url: https://github.com/chatterino/chatterino2
 edit_uri: ""
+markdown_extensions:
+  - toc:
+      permalink: true
```
Comment (alazymeme) [inline mkdocs.yml:21]: This adds anchors for quick url referencing.
image

Like this:
image

File: mkdocs.yml (hunk 1 above)
Comment (pajlada) [reply mkdocs.yml:9]: https://squidfunk.github.io/mkdocs-material/upgrading/#upgrading-from-5x-to-6x

**Review by pajlada — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**